app/forms/hyrax/forms/permission_template_form.rb

```diff
@@ -85,6 +81,18 @@ def remove_access!(permission_template_access)
         update_access(remove_agent: true)
       end
 
+      ##
+      # A bit of an analogue for a `belongs_to :source_model` as it crosses from Fedora to the DB
+      # @return [AdminSet, ::Collection]
+      # @raise [Hyrax::ObjectNotFoundError] when neither an AdminSet or Collection is found
+      # @note This method will eventually be replaced by #source which returns a Hyrax::Resource
+      #   object.  Many methods are equally able to process both Hyrax::Resource and
+      #   ActiveFedora::Base.  Only call this method if you need the ActiveFedora::Base object.
+      # @see #source
+      def source_model # rubocop:disable Rails/Delegate
```

Comment:
This is defined here instead of as a delegate to allow the note encouraging use of `#source` instead of `#source_model`.